ViewGenerator/tools/webpack/Plugins/ForkTsCheckerWebpackFormatterPlugin.ts

```diff
@@ -2,7 +2,7 @@ import ForkTsCheckerWebpackPlugin from "fork-ts-checker-webpack-plugin";
 import { Compiler } from "webpack";
 import { customErrorFormatter, getCurrentDirectory } from "./Utils";
 
-export default class ForkTsCheckerWebpackFormatterPlugin extends ForkTsCheckerWebpackPlugin{
+export default class ForkTsCheckerWebpackFormatterPlugin extends ForkTsCheckerWebpackPlugin {
```

Comment:
I think this is not needed anymore, the ForkTsChecker re/added this api

Reply:
good catch, it was not even used now... Removed the file